docs/05-reference/support-matrix.mdx

```diff
@@ -2,8 +2,41 @@
 title: Support Matrix
 ---
 
+## Inputs
```

Comment:
WDYT of better separating this doc overall between "on demand" (this) and "Livestream" (the ingest support)?

Reply:
addressed in newest commit